### ISSUE TITLE
/usr/bin/xcode-select -print-path does not ensure that the CLTools have actually been installed

### ISSUE THREAD

**Issue by severi** [elliotweiser/ansible-osx-command-line-tools#8]
Apparently the current check for testing whether command-line-tools have already been installed is not working.

```
- name: Check that command line tools are installed
  command: /usr/bin/xcode-select -print-path
  register: xcode_select_res
  ignore_errors: true
  changed_when: false

...
when: xcode_select_res.stdout|length == 0
```

Currently the check passes if XCode has been installed, but the CLTools haven't. 

Proper way for defining whether the command-line-tools have actually been installed would be running the following command:
`pkgutil --pkg-info=com.apple.pkg.CLTools_Executables`
which fails if the CLTools are not available.

**Comment by elliotweiser** [comment]
@severi Thanks for the heads up! I'll a take a look later this evening.

**Comment by elliotweiser** [comment]
@severi Out of curiosity, what's your target OS X version?

**Comment by severi** [comment]
The target host  is el capitan. I installed the xcode (8.2.1) using the .xip-file that can be downloaded from apple's developer portal. Not sure if there is a difference compared to installing it through the appstore. Let me know if you need more info :)

**Comment by elliotweiser** [comment]
@severi Your observation is correct. I didn't take into account that Xcode was installed after the command-line tools, which is a totally nontrivial, (non)edge-case. PR in progress!

**Comment by elliotweiser** [comment]
@severi Please check out #9 and let me know if you think I've missed anything. Unfortunately, using `pkgutil` only went so far because its receipt doesn't live in `/var/db/receipts` or `/Library/Receipts` like "normal" packages-- it lives in `/System/Library/Receipts`, which is protected by Apple System Integrity Protection (SIP). Since removing the Command Line Tools directory doesn't modify the package receipt and `pkgutil --forget` relies on being able to modify the receipt file, I had to think of something else.

Let me know if you have any suggestions. Otherwise, I'll try to get this merged by EOD.

**Comment by elliotweiser** [comment]
Fix will be available in version 1.2.0, which Travis CI should release some time tonight.
